home/audio.asm

```diff
@@ -50,8 +50,6 @@ UpdateSound::
 
 _LoadMusicByte::
 ; wCurMusicByte = [a:de]
-GLOBAL LoadMusicByte
```

Comment:
This was here because `BANK()` used to not flag labels for import. This was fixed a while ago.